api/controllers/zones.go

```diff
@@ -0,0 +1,39 @@
+package controllers
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/PedroChaparro/loomies-backend/interfaces"
+	"github.com/PedroChaparro/loomies-backend/models"
+	"github.com/gin-gonic/gin"
+)
+
+func HandleNearGyms(c *gin.Context) {
+
+	bodyCoord := interfaces.Coordinates{}
+
+	if err := c.BindJSON(&bodyCoord); err != nil {
+		fmt.Println(err)
+		c.AbortWithStatusJSON(http.StatusBadRequest, gin.H{"message": "Bad request"})
+		return
+	}
+
+	nearGyms, err := models.GetNearGyms(bodyCoord.Latitude, bodyCoord.Longitude)
+
+	if err != nil {
+		fmt.Println(err)
+		c.AbortWithStatusJSON(http.StatusInternalServerError, gin.H{"message": "Internal server error"})
+		return
+	}
+
+	if nearGyms == nil {
```

Comment:
1. This condition is upside down, when it founds Gyms near the user coordinates respond with a "Nothing found here" message.

2. If there are no gyms it should respond with an empty array instead of null, just to preserve the types (array).

Reply:
@SilviaPabon I notice that you remove the http.StatusNotFound condition, I think it's important to keep it.

Reply:
@PedroChaparro Solved!